### PR TITLE
Alt Lang pages: Added ability to use this feature of `sitemap.js`

### PR DESCRIPTION
**Pull request by woodyrew**
Includes test and description in README.md

### CODE REVIEW

**Comment by woodyrew** [comment]
_Note_: This doesn't conflict with #19

**Comment by ashishsc** [comment]
Thanks for the PR we will look at this soon!

File: README.md [hunk 1]
```diff
@@ -68,6 +68,12 @@ Change the output file for the sitemap.
 
 Add a lastmodified date to the sitemap. Should be a Date object and can be passed through the Javascript API or the frontmatter.
 
+##### links
+
+* `optional`
+
+Allows you define which field to use [for alternative language pages](https://github.com/ekalinin/sitemap.js#example-of-indicating-alternate-language-pages). This accepts nested properties in dot notation via [lodash.get](https://lodash.com/docs#get).
```
Comment (njbraun) [inline README.md]: Perhaps change to "Allows you to define alternate language pages"

**Comment by njbraun** [comment]
Once this and the other PR get merged in, I'll update version and publish to npmjs repo.

**Comment by woodyrew** [comment]
@njbraun Altered as requested and rebased to fix conflicts.

**Pull request merged**